rules/windows/powershell/powershell_script/posh_ps_namespace_X509Certificates.yml

```diff
@@ -0,0 +1,29 @@
+title: PowerShell X509Certificate Namespace in ScriptBlock
+id: 34949d3a-0684-4300-9ab1-2a50daf920fb
+status: experimental
+description: Detecting Hoaxshell reverse-shell (HTTPS Payload) which is using System.Security.Cryptography.X509Certificates namespace in the payload.
+references:
+    - https://github.com/t3l3machus/hoaxshell
```

Comment:
I think it's better to add the script permalink from the repo instead of the full repo.

https://github.com/t3l3machus/hoaxshell/blob/09b134379388712887de41bda5e41b12e6a56fd5/https_payload.ps1